### DISCUSSION TITLE
The GPU power consumption is locked at 35w after the screen is automatically turned off

### DISCUSSION THREAD

**Discussion by oriki23** [seerge/g-helper#2403] [Q&A]
![QQ图片20240408121025](https://github.com/seerge/g-helper/assets/166352756/387f604c-082b-4257-9475-7999f9327e58)
![屏幕截图 2024-04-08 121135](https://github.com/seerge/g-helper/assets/166352756/5b0a2447-5bfd-4167-b81f-42cfa7b22744)

**Comment by oriki23** [comment]
[Uploading log.txt…]()

**Reply by seerge** [reply]
@oriki23 G-Helper is not setting any of those limits and can't. You need to clearly explain your problem and upload all requested debug information as it's explained [in issue templates](https://github.com/seerge/g-helper/issues/new/choose)
